src/Screens/FeedPage/FeedPage.js

```diff
@@ -0,0 +1,348 @@
+                })}
+          </div>
+        </TabPanel>
+        <TabPanel value={foodCategory} index={7}>
+          <div className={classes.listContainer}>
+            {restaurants &&
+              restaurants
+                .filter((restaurant) => {
+                  return restaurant.category === "Mexicana";
+                })
+                .map((restaurant) => {
+                  return (
+                    <RestaurantCard
+                      key={restaurant.id}
+                      id={restaurant.id}
+                      name={restaurant.name}
+                      description={restaurant.description}
+                      address={restaurant.address}
+                      shipping={restaurant.shipping}
+                      deliveryTime={restaurant.deliveryTime}
+                      logoUrl={restaurant.logoUrl}
+                      category={restaurant.category}
+                    />
+                  );
+                })}
+          </div>
+        </TabPanel>
+        <TabPanel value={foodCategory} index={8}>
+          <div className={classes.listContainer}>
+            {restaurants &&
+              restaurants
+                .filter((restaurant) => {
+                  return restaurant.category === "Petiscos";
+                })
+                .map((restaurant) => {
+                  return (
+                    <RestaurantCard
+                      key={restaurant.id}
+                      id={restaurant.id}
+                      name={restaurant.name}
+                      description={restaurant.description}
+                      address={restaurant.address}
+                      shipping={restaurant.shipping}
+                      deliveryTime={restaurant.deliveryTime}
+                      logoUrl={restaurant.logoUrl}
+                      category={restaurant.category}
+                    />
+                  );
+                })}
+          </div>
+        </TabPanel>
+        <TabPanel value={foodCategory} index={9}>
+          <div className={classes.listContainer}>
+            {restaurants &&
+              restaurants
+                .filter((restaurant) => {
+                  return restaurant.category === "Sorvetes";
+                })
+                .map((restaurant) => {
+                  return (
+                    <RestaurantCard
+                      key={restaurant.id}
+                      id={restaurant.id}
+                      name={restaurant.name}
+                      description={restaurant.description}
+                      address={restaurant.address}
+                      shipping={restaurant.shipping}
+                      deliveryTime={restaurant.deliveryTime}
+                      logoUrl={restaurant.logoUrl}
+                      category={restaurant.category}
+                    />
+                  );
+                })}
+          </div>
+        </TabPanel>
+        <Footer />
+      </div>
+    </ThemeProvider>
```

Comment:
Esse código ficou beeeem grande. Mais de 300 linhas. Quando for assim, pensem em componentizar mais o código.